clang/lib/StaticAnalyzer/Checkers/WebKit/UncountedCallArgsChecker.cpp

```diff
@@ -56,12 +62,16 @@ class UncountedCallArgsChecker
       bool TraverseClassTemplateDecl(ClassTemplateDecl *Decl) {
         if (isRefType(safeGetName(Decl)))
           return true;
-        return RecursiveASTVisitor<LocalVisitor>::TraverseClassTemplateDecl(
-            Decl);
+        return Base::TraverseClassTemplateDecl(Decl);
+      }
+
+      bool TraverseDecl(Decl *D) {
```

Comment:
With this blanket "let's catch every decl" code it may catch a few very weird decls here and there (eg `VarDecl`, `BlockDecl`, etc). I don't know what the exact consequences of that look. They're probably benign. But it's probably easier to stick to `FunctionDecl` and `ObjCMethodDecl`.

Reply:
Ah, right, I'd add if for `FunctionDecl` and `ObjCMethodDecl`.